src/utils/parser/getOutOfScopeLocations.js

```diff
@@ -16,17 +16,17 @@ import { traverseAst } from "./utils/ast";
  * Returns all functions (declarations, expressions, arrows) for the given
  * source
  */
-function findFunctions(source: SourceText) {
+function findSymbols(source: SourceText) {
```

Comment:
This `findSymbols/findFunctions` function seems odd to me. Couldn't we reuse this [getSymbols](https://github.com/devtools-html/debugger.html/blob/master/src/utils/parser/getSymbols.js#L55) function and modify it to extract comments?